lib/aem/mapper.js

```diff
@@ -0,0 +1,26 @@
+import queryGraphQL from '../../graphql/client'
+import getDashboardPage from '../../graphql/queries/dashboardQuery.graphql'
+
+export default async function MapDashboard() {
+  try {
+    const aemContent = await queryGraphQL(getDashboardPage).then((result) => {
+      return result
+    })
+    const returnedQuery = aemContent.data.alphaSCHPageByPath.item
+    return {
+      en: {
+        pageName: returnedQuery.scPageNameEn,
+        pageTitle: returnedQuery.scTitleEn,
+      },
+      fr: {
+        pageName: returnedQuery.scPageNameFr,
+        pageTitle: returnedQuery.scTitleFr,
+      },
+    }
+  } catch (e) {
+    return {
```

Comment:
**Blocking** - Returning an object like this (one with a format that doesn't match the expected result) would cause the application to crash at runtime. I suspect if AEM fails we won't have a backup for content, so we'd want to redirect to a 500 error page and log the error.

Reply:
The Dashboard page should be responsible for redirecting when an error is returned that can't be handled